src/Stripe/Services/Customers/StripeCustomerCreateOptions.cs

```diff
@@ -28,6 +28,9 @@ public class StripeCustomerCreateOptions
         [JsonProperty("plan")]
         public string PlanId { get; set; }
 
+        [JsonProperty("tax_percent")]
+        public decimal? TaxPercent { get; set; }
```

Comment:
Stripe Internal Documentation, the guys from Support told me that because without this I need two API calls to crate a customer + subscription + tax percent. With this property it can be all done in one call.